src/output/json.rs

```diff
@@ -90,12 +90,12 @@ pub(crate) fn json_output(
     mut out: impl std::io::prelude::Write,
 ) -> std::io::Result<()> {
     let benchmark_results: JsonValue = JsonValue::Object(tree.into_iter().map(from).collect());
-    let pretty = serde_json::to_string_pretty(&json!(
+    let pretty = serde_json::to_string(&json!(
```

Comment:
the pretty printing was quite redundant, given 99% of the time just going to input to some scripts😂, easy trade for jsonlines